includes/Core/User/User.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\User\User
+ *
+ * @package   Google\Site_Kit\Core\User
+ * @copyright 2024 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\User;
+
+use Google\Site_Kit\Core\Storage\User_Options;
+
+/**
+ * Class for handling audience settings rest routes.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class User {
```

Comment:
The `User` name here is a bit misleading. This class is not about user, it is about audience segmentation settings that user has. Let's rename it to be `Audience_Segmentation` instead.

We can still have this `User` class but its purpose should be to instantiate special matters like this audience segmentation one. So, the `User` class instantiate and registers the `Audience_Segmentation` class that instantiates and registers `Audience_Settings` and rest controller.

Reply:
THanks. I updated class names and their usage